### DISCUSSION TITLE
Elsa Dashboard

### DISCUSSION THREAD

**Discussion by rmanjusha** [elsa-workflows/elsa-core#5371] [Q&A]
I am using Elsa Studio 3.1.0  and my services has dashboard module added using builder.Services.AddDashboardModule();

But studio still shows Dashboard page empty . Am I missing any configuration ?

**Comment by glime-ai** [comment]
To troubleshoot the issue with Elsa Studio showing an empty Dashboard page, even though you have added the dashboard module using `builder.Services.AddDashboardModule()`, you should check the following:

1. Ensure that you have correctly set up the Elsa Server and that it is running. Elsa Studio needs to connect to the Elsa Server to display workflows and other information.

2. Verify that the version of Elsa Studio you are using is compatible with the version of Elsa Server. Incompatibilities between versions can cause issues like the one you're experiencing.

3. Check the browser console for any errors that might indicate what is going wrong. Common issues could be related to JavaScript errors, failed network requests, or issues with loading necessary resources.

4. Make sure that you have configured the necessary services and middleware in your `Program.cs` or `Startup.cs` file. This includes setting up authentication, authorization, CORS policies, and the Elsa API endpoints.

5. If you are using Blazor Server, ensure that you have followed the instructions for setting up Elsa Studio with Blazor Server, as outlined in the documentation.

6. Review the GitHub issues for Elsa to see if others have encountered similar problems and if there are any suggested solutions or workarounds.

7. If you have recently upgraded Elsa Studio or Elsa Server, make sure that you have followed the migration guide and updated your configuration accordingly.

If you have checked all of the above and the issue persists, you may want to reach out to the Elsa community or support for further assistance. Providing detailed information about your setup, including any error messages and the steps you have taken to configure Elsa Studio, will help in diagnosing the problem.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
